src/IO/Lz4DeflatingWriteBuffer.cpp

```diff
@@ -125,6 +125,8 @@ void Lz4DeflatingWriteBuffer::nextImpl()
         out->position() = out->buffer().begin();
         throw;
     }
+    out->next();
+    out_capacity = out->buffer().end() - out->position();
```

Comment:
These two lines though seem a little unnecessary.
```suggestion
```

Reply:
If I remember correctly, if I remove these lines, it will break test. Will check it now

Reply:
Yes, removing of these lines breaks it...